mod-tcp-goodput.md

```diff
@@ -23,25 +28,136 @@ simple data exchange it has some novel features.
 - Selectable congestion control algorithm
 - Parallel Workers (thread support)
 
+## Name
+
+This module is standardized with the name:
+
+```
+tcp-goodput
+```
+
+## Messages
+
+### Info Reply
+
+`tcp-goodput` has no additional information for the client. The `tcp-goodput`
```

Comment:
"E.g. the "tcp-goodput"
   module has a section which specifies allowed values for the tcp-goodput
  module."

contrary statements: 
https://github.com/protocollabs/maparo-std/pull/1/files#r223979114

Reply:
additional fields. additional. seq-rq is standard and always required.